app/util/smtp.py

```diff
@@ -0,0 +1,29 @@
+import smtplib
+import sys
+from six import reraise as raise_
+
+
+class SMTP(object):
+    def __init__(self, host, port, user, password, timeout = 5):
+        self.conn = None
+        self.host = host
+        self.port = port
+        self.user = user
+        self.password = password
+        self.timeout = timeout
+
+    def send_message(self, mail_from, mailto, message):
+        try:
+            self.conn = smtplib.SMTP(host=self.host, port=self.port, timeout=self.timeout)
+            self.conn.ehlo()
+            self.conn.starttls()
+            self.conn.login(self.user, self.password)
```

Comment:
so your application is using let's say your gmail account to actually send messages, can your gmail account handle 100 million messages a day? how many gmail accounts do we need to support this kind of traffic :)

Reply:
No, of course gmail account cannot handle such a huge amount of messages, because google blocks it for spamming. I guess for this task  we need own multiple smtp  servers. Or maybe sending messages w/o smtp relays, i.e. get list of mx records for recipients mail domains, connect to each host and pass message directly